lib/langchain/data.rb

```diff
@@ -12,6 +12,7 @@ class Data
     def initialize(data, options = {})
       @source = options[:source]
       @data = data
+      @chunker_klass = options[:chunker] || Langchain::Chunker::Text
```

Comment:
What're your thoughts on spelling out the method parameters here?
```ruby
def initialize(data, source:, chunker: Langchain::Chunker::Text)
```

I know this `options = {}`-style parameter is everywhere in this codebase but I've gotten to really dislike it. It makes sense when the it's an interface that can't be controller and you're just passing `options` along, something like:
```ruby
def our_method(options = {})
   ThirdPartyService.method(**options)
```
but in the cases where we know exactly what the interface is -- it makes sense to explicitly define it. It's also much better to document exactly what parameters can be passed in.

Reply:
I like that idea. I followed the convention and the path of least resistance, but it felt like _options_ was getting overloaded. This is especially true in `Loader` and `Vectorsearch#add_data` where the options are now passed to multiple classes they initialize.  Would you want that same specific parameter for those as well?

I'm happy to rework the PR

Reply:
@spikex Preferably, yes :)